mirror/required-images.txt

```diff
@@ -44,6 +46,7 @@ quay.io/kubernetes-ingress-controller/nginx-ingress-controller:0.20.0
 
 # ETCD v3 
 k8s.gcr.io/etcd:3.2.18
+k8s.gcr.io/etcd:3.2.24
```

Comment:
can you provide any URL or screenshot of this?

Reply:
Actually the kops standard configure was 3.2.24. Actually it's not a problem, because in the cluster configuration you can set version 3.2.18. You can ignore this part of the PR and I will raise a new if I have more information after a fresh installation. ;)

Reply:
If you can find any config in the upstream kops that points the version to `3.2.24`, please past the URL here before I can merge this PR.

thanks.

Reply:
I found only that the 3.2.18 is the default and some advice to use 3.2.24. But it should be "the default 3.2.18"

Reply:
https://github.com/kubernetes/kops/blob/master/pkg/model/components/etcd.go